rasa/shared/core/events.py

```diff
@@ -489,23 +525,34 @@ def _entity_string(self):
         return ""
 
     def as_story_string(self, e2e: bool = False) -> Text:
```

Comment:
should here be deprecation warning?

Reply:
I'd deprecate MD as a whole. I also re-started the discussion on this in the `product` channel in Slack.